lindera/src/token_filter/japanese_katakana_stemming.rs

```diff
@@ -0,0 +1,249 @@
+use std::num::NonZeroUsize;
+
+use lindera_core::token_filter::TokenFilter;
+use serde::{Deserialize, Serialize};
+
+use crate::{error::LinderaErrorKind, LinderaResult, Token};
+
+const DEFAULT_MIN: usize = 3;
+const DEFAULT_HIRAGANA_KATAKANA_PROLONGED_SOUND_MARK: char = '\u{30FC}';
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+pub struct JapaneseKatakanaStemmingTokenFilterConfig {
+    min: Option<NonZeroUsize>,
```

Comment:
Can we have just `usize` here, instead of unwrapping the value on every `apply()` call?

Reply:
We can remove Option, but I'd like to continue using NonZeroUsize so that min cannot be set to 0.
How about this?
https://github.com/lindera-morphology/lindera/pull/237/commits/5d4c96751ff0ea319b4bda508efa09268cfca4b7